### ISSUE TITLE
Model weights for the parikh entailment and sentence similarity siamese cnn

### ISSUE THREAD

**Issue by ericzhao28** [explosion/spaCy#997]
Hey,

Are pre-trained model weights from the Keras Parikh Entailment example publicly available? If not, would it be possible to add them to the example for those who'd like to play around with the example without needing to train fully themselves?(https://github.com/explosion/spaCy/tree/master/examples/keras_parikh_entailment). I know Honnibal hasn't rerun the full experiment since refactoring things, but are the model weights for the pre-refactored code still available?

Likewise, I was also curious as to whether the model weights used for the Siamese CNN model demo at https://demos.explosion.ai/similarity are available (detailed here: https://explosion.ai/blog/supervised-similarity-siamese-cnn). 

Eric

**Comment by dnm1977** [comment]
I second this suggestion -- viz., making pre-trained weights publicly.  It would allow us to quickly experiment with these models for other purposes where advancing the state of the art in RTE (or insert-domain-of-model-here) isn't our concern.  Bonus: there will be an exact means of running a baseline of record for the community at large.  (Also training with GPUs is expensive.)

**Comment by ines** [comment]
Merging this with #1445!

> Likewise, I was also curious as to whether the model weights used for the Siamese CNN model demo at https://demos.explosion.ai/similarity are available (detailed here: https://explosion.ai/blog/supervised-similarity-siamese-cnn).

We're currently working on a new version of the model that will hopefully achieve much better accuracy than the previous one. In the meantime, we've replaced the similarity demo with a version showing spaCy's `doc.similarity()` method. See #1406 for more details on this.
